.github/workflows/main.yml

```diff
@@ -8,8 +8,11 @@ on:
   pull_request:
     branches: ["main"]
 
+  workflow_dispatch:
```

Comment:
Are you sure to use workflow_dispatch in current situation? Please read documentation about this and give your opinion, why we need to use it :)
Link to docs: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch